src/views/List.css

```diff
@@ -98,19 +98,48 @@ section {
 	display: flex;
 	flex-direction: column;
 	align-items: center;
+	padding-top: 2em;
+	padding-bottom: 2em;
 }
 
 /* List Item */
+.listContainer {
+	width: 100%;
+	border: black 3px solid;
```

Comment:
Does the container itself need a border? If each row has a border, the border is taken care of, and it won't be doubled up.